messenger/App.js

```diff
@@ -0,0 +1,57 @@
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { NavigationContainer } from '@react-navigation/native';
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import ConversationsPage from './Pages/ConversationsPage';
+import SettingsPage from './Pages/SettingsPage';
+import FriendsPage from './Pages/FriendsPage';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import { Provider } from 'react-redux';
+import store from './store/store';
```

Comment:
Alphabetize?

Reply:
Retweet on alphabetize.  I usually alphabetize by separating curly braces and having any "* as" at the top.  So as an example:

import * as Object from './folder/file.file'
import ABC from './randomfile1'
import BCD from './anotherrandomfile'
import { ABCD } from './abc'
import { XYZ } from './folder1/folder2/hi.txt'

Reply:
You could also alphabetize independent of whether curly braces are used